src/languages/es.js

```diff
@@ -302,6 +302,11 @@ export default {
         offline: 'Desconectado',
         syncing: 'Sincronizando',
     },
+    timezonePage: {
+        timezone: 'Zona horaria',
+        isShownOnProfile: '',
+        getLocationAutomatically: '',
```

Comment:
```suggestion
        isShownOnProfile: 'Tu zona horaria se muestra en tu perfil.',
        getLocationAutomatically: 'Detecta tu ubicación automáticamente.',
```

But different suggested copy here: https://expensify.slack.com/archives/C21FRDWCV/p1667233155627049?thread_ts=1667231566.616539&cid=C21FRDWCV

Reply:
Looks good, I'd say we should either bump in the thread for clarification or add these so there is some translation at least.